trollmoves/movers.py

```diff
@@ -357,6 +357,43 @@ def move(self):
 
     def copy(self):
         """Upload the file."""
+        from scp import SCPError
+
+        retries = 3
+        success = False
+        while retries > 0:
+            retries -= 1
+
+            try:
+                scp = self._get_scp_client()
+                scp.put(self.origin, self.destination.path)
+                success = True
+                break
```

Comment:
Do you need the break here if you break on success further down?

Reply:
I was already pretty tired when writing that, so probably not :sweat_smile: